### PR TITLE
Add podspec file

### PR DESCRIPTION
**Pull request by leninlin**

### CODE REVIEW

File: WatermelonDB.podspec [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+   s.name         = "WatermelonDB"
+   s.version      = package["version"]
+   s.summary      = package["description"]
+   s.description  = package["description"]
+   s.homepage     = package["homepage"]
+   s.license      = package["licence"]
+   s.author       = { "author" => package["author"] }
+   s.platforms    = { :ios => "9.0", :tvos => "9.0" }
+   s.source       = { :http => 'file:' + __dir__ + '/'  }
+   s.source_files = "native/ios/**/*.{h,m,swift}"
```
Comment (radex) [inline WatermelonDB.podspec:15]: i think FMDB/ needs to be skipped

Comment (leninlin) [reply WatermelonDB.podspec:15]: No need. But needs to remove the dependence below

**Comment by leninlin** [comment]
@radex 
I fixed the dependencies. Can you merge this?

**Comment by leninlin** [comment]
related issue #279

**Comment by radex** [comment]
ah, one more thing @leninlin -- please add a line to CHANGELOG about this :)

**Comment by leninlin** [comment]
Done ;)

**Comment by leninlin** [comment]
Any updates?

**Comment by radex** [comment]
I'll be looking into it soon - sorry bout the delay.

**Comment by todorone** [comment]
@radex Currently WatermelonDB installation is broken on iOS since 0.61, which is almost year away...😸 
Is there anything could be done to push this PR forward?
Thanks.

**Comment by radex** [comment]
@todorone not true! Works just fine, just manually integrated, per instructions, not via CocoaPods :) 

I want to merge this PR, but have to test it first - didn't have time recently. I plan on doing so sometime soon, as I'd like to migrate my projects to CocoaPods-based integration too.

**Comment by todorone** [comment]
@radex Please, don't take it as an offence or pressure. Just wanted to say that it's been pretty long ago since CocoaPods was established as default dependencies resolver for React Native and auto-linking works automatically for all more/less maintained libraries so current WatermelonDB installation flow filters out most of developers want to try this amazing project. Thanks for all the hard work you put into it and sorry if i sounded harsh.

**Comment by radex** [comment]
> Thanks for all the hard work you put into it and sorry if i sounded harsh.

you didn't - and no offense taken. you just said it's "broken", and it's not -- just, like you said, using 🍉 requires some effort to set up ;)

**Comment by sidferreira** [comment]
@radex coming from https://github.com/Nozbe/WatermelonDB/pull/570, it seems you still need to test it. If you are OK, I can try it in my repo. Based on PR's description I expect it to be a drop-in replacement.

**Comment by sidferreira** [comment]
@leninlin if you don't mind, can you create a branch (instead of using master) to make it easier to test?

**Comment by radex** [comment]
@sidferreira please do :)

**Comment by radex** [comment]
I'm looking at this PR right now. Sadly, it does not compile after JSI changes. Looking into it.

**Comment by radex** [comment]
Unfortunately, more changes were required, and it took me an hour or two of (frankly wasted) time… but, on the bright side, it should work now!

https://github.com/Nozbe/WatermelonDB/pull/731

**Pull request closed without merging**